package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "novuhq",
+  "name": "repo",
```

I'm typically referring to this particular `package.json` as the "repository root package.json", I think `name: "root"` serves to support this story well. `repo` additionally feels a bit too Git-like IMO. Wdyt?

```suggestion
  "name": "root",
```